docs/sphinx/reference-libobs-graphics-graphics.rst

```diff
@@ -997,6 +997,60 @@ Texture Functions
 
 ---------------------
 
+.. type:: enum gs_dmabuf_flags
+
+   DMA-BUF capabilities:
+
+   - GS_DMABUF_FLAG_NONE
+   - GS_DMABUF_FLAG_IMPLICIT_MODIFIERS_SUPPORTED  - Renderer supports implicit modifiers
```

Comment:
I'd rather use `GS_DMABUF_FLAG_SUPPORTS_IMPLICIT_MODIFIERS` instead - not a fan of using past tense in enums